crypto/cipher_extra/cipher_test.cc

```diff
@@ -525,6 +525,53 @@ TEST(CipherTest, SHA1WithSecretSuffix) {
   }
 }
 
+TEST(CipherTest, SHA256WithSecretSuffix) {
```

Comment:
Is this test actually functional (i.e. does it do any CT checking)?

Reply:
`SHA256WithSecretSuffix` CT was checked in `CryptoAlg-1067`. Basically, some constant code is changed to non constant and the valgrind can detect the change. e.g.
```
[ RUN      ] CipherTest.SHA256WithSecretSuffix
==3891== Conditional jump or move depends on uninitialised value(s)
==3891==    at 0x566BA6: constant_time_is_zero_w (internal.h:388)
==3891==    by 0x566BD7: constant_time_eq_w (internal.h:405)
==3891==    by 0x566BFC: constant_time_eq_8 (internal.h:411)
```